test/unit/config/index.test.js

```diff
@@ -0,0 +1,18 @@
+describe('main config', () => {
```

Comment:
With all of these config test (index, cookie, cert-request-queue, shared-sb-config, blob) should there be a test for valid config? Currently just checking invalid config is thrown but not testing valid config doesn't throw

Reply:
config is currently covered through all the other tests, it is basically called on every route call as far as I understand

Reply:
took the approach of only adding extra tests for branches that weren't covered.  I think this will give us a lot of unit tests that could be utilised in future.